docs/pages/dsCompositeIn/ds-compositeIn-L3-Low-Level-TestSpec.md

```diff
@@ -0,0 +1,198 @@
+# Device Settings Composite Input L3 Low Level Test Specification and Procedure Documentation
+
+## Table of Contents
+
+- [Overview](#overview)
+- [Acronyms, Terms and Abbreviations](#acronyms-terms-and-abbreviations)
+- [References](#references)
+- [Level 3 Test cases High Level Overview](#level-3-test-cases-high-level-overview)
+- [Level 3 Python Test](#level-3-python-test-cases-high-level-overview)
+
+## Overview
+
+This document describes the L3 Low Level Test Specification and Procedure Documentation for the Device Settings Composite Input module.
+
+### Acronyms, Terms and Abbreviations
+
+- `HAL`    - Hardware Abstraction Layer
+- `API`    - Application Programming Interface
+- `L2`     - Level 2 Testing
+- `L3`     - Level 3 Testing
+- `DS`     - Device Settings
+- `DUT`    - Device Under Test
+- `NA`     - Not Applicable
+- `RAFT`   - Rapid Automation Framework for Testing
+
+### References
+
+- `High Level Test Specification` - [ds-compositeIn-High-Level_TestSpec.md](https://github.com/rdkcentral/rdk-halif-test-device_settings/blob/main/docs/pages/ds-compositeIn-High-Level_TestSpec.md)
```

Comment:
you can update link like this 
so if the version changes no problem 

`High Level Test Specification` - [ds-compositeIn-High-Level_TestSpec.md](ds-compositeIn-High-Level_TestSpec.md)

Reply:
done